src/components/Header/SideNav/CloseButton.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import styled from 'styled-components'
+import remcalc from 'remcalc'
+
+import close from '../../../images/close.svg'
+import ButtonAsIconWrapper from '../../Common/ButtonAsIconWrapper'
+import outlineStyles from '../outlineStyles'
+
+const CloseIcon = styled.img`
+  fill: ${props => props.theme.colors.white};
```

Comment:
I don't think `fill` will work here since this is an img (it would if we were using an `svg` tag)

Reply:
:woman_facepalming: